src/palimpzest/prompts.py

```diff
@@ -56,7 +61,7 @@
 
 FILTER CONDITION: {filter_condition}
 
-ANSWER: 
+ANSWER:
```

Comment:
I would add back the space after the `:` here, as some papers have reported that this small change to a prompt improves model performance.